core/envelope-sim/src/main/java/fr/sncf/osrd/envelope/DriverBehaviour.java

```diff
@@ -0,0 +1,17 @@
+package fr.sncf.osrd.envelope;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@SuppressFBWarnings("URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD")
+public class DriverBehaviour {
+    public double acceleratingPostponementOffset = 50;
+    public double brakingAnticipationOffset = 100;
+
+    public DriverBehaviour(double acceleratingPostponementOffset, double brakingAnticipationOffset) {
+        this.acceleratingPostponementOffset = acceleratingPostponementOffset;
+        this.brakingAnticipationOffset = brakingAnticipationOffset;
+    }
+
+    public DriverBehaviour() {
+    }
```

Comment:
you mean (100,50)
If the default values are not going to change later and they will remain fix, this solution could work.
But if the default values will change, it will be harder to maintain the code and change all declarations used with default values.

Reply:
I did miss the (100, 50) above.

But that's kind of my point: the default constructor is here to set the default values, it would be easier to follow when the default values are specified in this constructor.